src/ILAMB/Variable.py

```diff
@@ -128,8 +266,11 @@ def __init__(self,**keywords):
             assert variable_name is not None
             t0 = keywords.get("t0",None)
             tf = keywords.get("tf",None)
+            z0 = keywords.get("z0",None)
```

Comment:
I had the time range in the Variable constructor because time is the leading dimension in netcdf files and only reading in what you need is a huge savings in memory I/O. If these depth bounds are in the constructor, why not also lat and lon?

Reply:
I see. Should I remove the lines 269-270 z0 & zf statements and the z0 & zf statements input into il.FromNetCDF4, and test if things work?

Reply:
Let's just leave it.